pkg/platform/runtime/buildplan/buildplan.go

```diff
@@ -48,6 +56,33 @@ func NewMapFromBuildPlan(build *model.Build) (artifact.Map, error) {
 	return res, nil
 }
 
+func filterTerminals(build *model.Build) ([]*model.NamedTarget, error) {
```

Comment:
Add comments please. Reading the code this seems to be about filtering only the terminal relevant to our platform, might be good to name it something more indicative like `filterPlatformTerminals()`. Current naming is very generic.